extensions/ql-vscode/src/view/method-modeling/MethodModeling.tsx

```diff
@@ -1,9 +1,35 @@
 import * as React from "react";
+import { styled } from "styled-components";
+import { ModelingStatusIndicator } from "../model-editor/ModelingStatusIndicator";
+
+const Container = styled.div`
+  background-color: var(--vscode-peekViewResult-background);
+  padding: 0.3rem;
+  margin-bottom: 1rem;
+`;
+
+const Title = styled.div`
+  padding-bottom: 0.3rem;
+  font-size: 1.2em;
```

Comment:
I'm wondering if it would be better to use `large` etc. rather than a specific number. I see we have a mixture of these elsewhere in the codebase but it seems like a good idea to use more "standard" sizes.